### PR TITLE
remove memcache option

### PR DESCRIPTION
**Pull request by briskt**
### Removed
- Removed memcache option for session storage

https://itse.youtrack.cloud/issue/IDP-1041

### CODE REVIEW

File: development/idp-local/config/config.php [hunk 1]
```diff
@@ -806,20 +802,13 @@
...
-            ],
-        ],
-        [
-            [
-                'hostname' => $MEMCACHE_HOST2,
-                'port' => $MEMCACHE_HOST2_PORT,
-            ],
-        ],
-    ],
+//    'memcache_store.servers' => [
```
Comment (mtompset) [inline development/idp-local/config/config.php:805]: Why not /* */ comment around the outdated code?

Comment (jason-jackson) [reply development/idp-local/config/config.php:805]: I think doing that would make it too similar to comments detailing what config values do.

**Review by jason-jackson — APPROVED** [review]
I don't know remember if this is being used or not and didn't check that. Code looks fine.

File: development/idp-local/config/config.php (hunk 1 above)
Comment (briskt) [reply development/idp-local/config/config.php:805]: Not to mention that PhpStorm doesn't use /* */ when you press Ctrl-/

**Comment by briskt** [comment]
> I don't know remember if this is being used or not and didn't check that. Code looks fine.

Hasn't been used since the multiregion project. We decided to save money by using the database instead. No need to pay for Elasticache since we don't need the slight performance benefit in provides.

**Pull request merged**